core/common/src/Code.ts

```diff
@@ -34,16 +34,16 @@ export class Code implements CodeProps {
   /** the [CodeValue]($docs/bis/intro/codes.md#codevalue-property) of the Element */
   public value?: string;
 
-  constructor(val: CodeProps) {
-    this.spec = Id64.fromJSON(val.spec);
-    this.scope = JsonUtils.asString(val.scope, "");
-    this.value = JsonUtils.asString(val.value);
+  constructor(codeProps: CodeProps) {
+    this.spec = Id64.fromJSON(codeProps.spec);
+    this.scope = JsonUtils.asString(codeProps.scope);
+    this.value = JsonUtils.asString(codeProps.value?.trim());
   }
 
   /** Create an empty, non-unique code with no special meaning. */
   public static createEmpty(): Code { const id: Id64String = Id64.fromLocalAndBriefcaseIds(1, 0); return new Code({ spec: id, scope: id }); }
   public static fromJSON(json?: any): Code { return json ? new Code(json) : Code.createEmpty(); }
-  public getValue(): string { return this.value ? this.value : ""; }
+  public getValue(): string { return this.value ?? ""; }
```

Comment:
i agree with Paul, value should be a property. Then, deprecate this method.

Reply:
Done